packages/esm-active-visits-app/src/config-schema.ts

```diff
@@ -0,0 +1,10 @@
+import { Type, validators } from '@openmrs/esm-framework';
+export const configSchema = {
+  search: {
```

Comment:
Does not make any sense. Why does the active widgets MF have a config that has something to do with search?